### PR TITLE
Makes necromantic stones more customizable

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Adds a variable to necromantic stones, that determines to which species will be changed the target upon use

## Why It's Good For The Game

Makes this item more customizable by code and admin varedits

## Changelog

:cl:
code: Now species to which are changed necromantic stone targets are determinated by a variable rather then just skeleton species
/:cl:

### CODE REVIEW

File: code/modules/antagonists/wizard/equipment/artefact.dm [hunk 1]
```diff
@@ -220,6 +220,7 @@
 	w_class = WEIGHT_CLASS_TINY
 	var/list/spooky_scaries = list()
 	var/unlimited = 0
+	var/spec = /datum/species/skeleton
```
Comment (CapybaraExtravagante) [inline code/modules/antagonists/wizard/equipment/artefact.dm:223]: You're missing auto-doc for this var, and the var name isn't super descriptive. maybe do something like applied_species

Comment (Time-Green) [inline code/modules/antagonists/wizard/equipment/artefact.dm:223]: Use descriptive variable names and autodoc your comments  
https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md

**Review by Time-Green — CHANGES_REQUESTED** [review]

**Review by Time-Green — APPROVED** [review]

File: code/modules/antagonists/wizard/equipment/artefact.dm (hunk 1 above)
Comment (CapybaraExtravagante) [reply code/modules/antagonists/wizard/equipment/artefact.dm]: thats what i said copycat

Comment (Time-Green) [reply code/modules/antagonists/wizard/equipment/artefact.dm]: I hereby maintainer decree that I said it first and any evidence in the contrary is a github bug

**Comment by VladinXXV** [comment]
can't a guy just raise a family in peace?

**Comment by Blevruz** [comment]
oh no... nekomantic stone

**Pull request merged**